test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua

```diff
@@ -51,7 +51,7 @@ local hmi_result_code = {
 	{ result_code = "ABORTED", info = "" },
 	{ result_code = "IGNORED", info = "" },
 	{ result_code = "IN_USE", info = "" },
-	{ result_code = "DATA_NOT_AVAILABLE", info = "" },
+	{ result_code = "VEHICLE_DATA_NOT_AVAILABLE", info = "" },
```

Comment:
@istoimenova, 'VEHICLE_DATA_NOT_AVAILABLE' error code is not available in HMI_API. Instead of it there is 'DATA_NOT_AVAILABLE' one. Is there some kind of mapping between HMI and MOBILE result codes?